### PR TITLE
Added grading and units to the content type and to be pulled in through ...

### PR DESCRIPTION
**Pull request by cjwest**
...the feed.

### CODE REVIEW

File: stanford_courses.info [hunk 1]
```diff
@@ -7,12 +7,12 @@ project = stanford_courses
 dependencies[] = ctools
 dependencies[] = date
 dependencies[] = date_popup
-dependencies[] = features (>=2.0)
-dependencies[] = feeds (>=2.0-alpha8)
+dependencies[] = features
```
Comment (sherakama) [inline stanford_courses.info]: Please restore the version numbers.
Features when exporting does not maintain these. 
Pain in da butt.

File: stanford_courses.info [hunk 2]
```diff
@@ -120,4 +124,5 @@ features_exclude[field_instance][field_collection_item-field_s_image_info-field_
 features_exclude[field_instance][field_collection_item-field_s_image_info-field_s_image_credits] = field_collection_item-field_s_image_info-field_s_image_credits
 features_exclude[field_instance][field_collection_item-field_s_image_info-field_s_image_image] = field_collection_item-field_s_image_info-field_s_image_image
 features_exclude[field_instance][field_collection_item-field_s_image_info-field_s_image_source_info] = field_collection_item-field_s_image_info-field_s_image_source_info
-project status url = 'https://github.com/SU-SWS/stanford_courses'
+project path = sites/all/modules/stanford
```
Comment (sherakama) [inline stanford_courses.info]: Please remove this line. No need for project path and causes issues for others.

**Comment by sherakama** [comment]
Just two small comments. 
I did not test the functionality but syntactically it looks good.

File: stanford_courses.info (hunk 2 above)
Comment (jbickar) [inline stanford_courses.info]: Does Features do this automatically? I noticed this when cleaning up some other Features the other day.

Comment (sherakama) [inline stanford_courses.info]: When you export from the UI and have the export path field filled in with a value it does.

Comment (sherakama) [inline stanford_courses.info]: FU does not FWIK.

**Comment by cjwest** [comment]
Thanks Shea!
I made those changes.

You can see it in action at:
https://sites.stanford.edu/jsa-courses-test/node/179/edit

**Comment by sherakama** [comment]
Looks good. Thanks.

**Pull request merged**